me_unpack.py

```diff
@@ -293,10 +293,10 @@ def extract_code_mods(nm, f, soff):
 
 
 def decomp_lzma(compdata):
-    import subprocess, _subprocess
-    # hide the console window
-    si = subprocess.STARTUPINFO()
-    si.dwFlags |= _subprocess.STARTF_USESHOWWINDOW
+    import subprocess
+    ## hide the console window
+    #si = subprocess.STARTUPINFO()
```

Comment:
the commented out lines are necessary on Windows to avoid unneeded console windows when running the decompressor. Please add a check for `os.name=="nt"` or similar.